app/lib/common/toolkit/buttons/inline_text_button.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:acter/common/themes/acter_theme.dart';
+import 'package:flutter/material.dart';
+import 'dart:math' as math;
+import 'dart:ui' show lerpDouble;
+
+class ActerInlineTextButton extends TextButton {
```

Comment:
Do we require separate class for `ActerInlineTextButton`? How about using default `TextButton` with our required theme set?

Reply:
Because there are _two_ ways in which the TextButton is used and they differ in design. Take the first-run-feature-introduction-wizzard and its "Previous", "Next" and "Skip" buttons and then the underlined button as used here in this comments creation above. We can't have a unified design for both of them because we want them to look different.

I opted for keeping the current TextButton as is and instead create a custom version with underlining for this more specific case (though I intend to spread that one a bit more widely in the future). That's why this is also opting for different theme internally.